index.bs

```diff
@@ -721,6 +721,43 @@ following steps:
 
 </div>
 
+<h2 id="privacy-and-security-considerations">Privacy and Security Considerations</h2>
+
+This feature will likely need to store information about sites that have a user interaction, some 
```

Comment:
Can you look for language like "likely need" that was speculative in explainer but we can be more clear about now?  We can say we do store information.

Reply:
Done. The first sentence is the only example I could find.

Also added a Note to the Timers section as we discussed offline.